example/aks/main.tf

```diff
@@ -0,0 +1,114 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.44.0"
```

Comment:
Suggest updating that to the minimum version required for the AKS UAI fix (I think it was 2.57).

And seeing as how we're in the examples section, use the "~>" operator like so: `version = "~> 2.57"`